README.md

```diff
@@ -188,11 +188,11 @@ cargo run -- build crate --local /path/to/source
 cargo run -- database add-directory <DIRECTORY> [PREFIX]
 
 # Updates github stats for crates.
-# You need to set CRATESFYI_GITHUB_USERNAME, CRATESFYI_GITHUB_ACCESSTOKEN
+# You need to set CRATESFYI_GITHUB_ACCESSTOKEN
 # environment variables in order to run this command.
 # Set DOCSRS_GITLAB_ACCESSTOKEN to raise the rate limit,
 # or leave it blank to fetch repositories at a slower rate.
```

Comment:
```suggestion
# or leave it blank to fetch these repositories at a slower rate.
```

Reply:
Hmm, I don't think this makes things more clear. I would probably just say
```suggestion
# or leave it blank to fetch them at a slower rate.
```
now that we've specified "GitLab" on the line above.